rules/openmpt.json

```diff
@@ -0,0 +1,115 @@
+{
+  "patterns": ["\\bopenmpt\\b"],
+  "dependencies": [
+    {
+      "packages": ["libopenmpt-dev"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "ubuntu"
+        },
+        {
+          "os": "linux",
+          "distribution": "debian"
+        }
+      ]
+    },
+    {
+      "packages": ["libopenmpt-devel"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "opensuse"
+        },
+        {
+          "os": "linux",
+          "distribution": "fedora"
+        }
+      ]
+    },
+    {
+      "packages": ["libopenmpt-devel"],
+      "pre_install": [
+        { "command": "yum install -y epel-release" }
+      ],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "centos"
+        },
+        {
+          "os": "linux",
+          "distribution": "rockylinux"
+        },
+        {
+          "os": "linux",
+          "distribution": "redhat"
+        }
+      ]
+    },
+    {
+      "packages": ["libopenmpt-dev"],
+      "constraints": [
+        {
+          "os": "linux",
+          "distribution": "alpine"
+        }
+      ]
+    }
+    {
```

Comment:
There's a syntax error here, and also I believe the `packages` will need to be combined or only the first one will be used:
```json
"packages": ["libopenmpt-devel", "portaudio-devel"],
```

Otherwise, this looks good, thanks!